src/health.h

```diff
@@ -111,262 +129,405 @@ typedef struct rrddimvar {
 // having as RRDSET.calculations the RRDCALC to be processed
 // next.
 
-#define RRDCALC_STATUS_REMOVED       -2
-#define RRDCALC_STATUS_UNDEFINED     -1
-#define RRDCALC_STATUS_UNINITIALIZED  0
-#define RRDCALC_STATUS_CLEAR          1
-#define RRDCALC_STATUS_RAISED         2
-#define RRDCALC_STATUS_WARNING        3
-#define RRDCALC_STATUS_CRITICAL       4
+#define RRDCALC_STATUS_REMOVED       -2 ///< Status is removed
+#define RRDCALC_STATUS_UNDEFINED     -1 ///< Status is undefined
+#define RRDCALC_STATUS_UNINITIALIZED  0 ///< Status is uninitialized
+#define RRDCALC_STATUS_CLEAR          1 ///< Status is clear 
+#define RRDCALC_STATUS_RAISED         2 ///< Status is raised
+#define RRDCALC_STATUS_WARNING        3 ///< Status is warning
+#define RRDCALC_STATUS_CRITICAL       4 ///< Status is critical
 
-#define RRDCALC_FLAG_DB_ERROR              0x00000001
-#define RRDCALC_FLAG_DB_NAN                0x00000002
+#define RRDCALC_FLAG_DB_ERROR              0x00000001 ///< Database eror occured
+#define RRDCALC_FLAG_DB_NAN                0x00000002 ///< Value from database is not a number
 /* #define RRDCALC_FLAG_DB_STALE           0x00000004 */
-#define RRDCALC_FLAG_CALC_ERROR            0x00000008
-#define RRDCALC_FLAG_WARN_ERROR            0x00000010
-#define RRDCALC_FLAG_CRIT_ERROR            0x00000020
-#define RRDCALC_FLAG_RUNNABLE              0x00000040
-#define RRDCALC_FLAG_NO_CLEAR_NOTIFICATION 0x80000000
+#define RRDCALC_FLAG_CALC_ERROR            0x00000008 ///< Calulation error occured
+#define RRDCALC_FLAG_WARN_ERROR            0x00000010 ///< Error level warning occured
+#define RRDCALC_FLAG_CRIT_ERROR            0x00000020 ///< error level error occured
+#define RRDCALC_FLAG_RUNNABLE              0x00000040 ///< ktsaou: Your help needed.
```

Comment:
`RRDCALC_FLAG_RUNNABLE` is used to cache the response of `rrdcalc_isrunnable()`. It is just an internal optimization for `health_main()`.